src/main/scala/org/zalando/spearheads/innkeeper/utils/HttpClient.scala

```diff
@@ -67,3 +78,14 @@ class AkkaHttpClient @Inject() (
     }
   }
 }
+
+private object ConnectionPoolFactory {
+
+  def apply(uri: Uri)(implicit actorSystem: ActorSystem, materializer: ActorMaterializer): Flow[(HttpRequest, Int), (Try[HttpResponse], Int), HostConnectionPool] = {
+
+    uri.scheme match {
+      case "shttp" | "https" => Http().cachedHostConnectionPoolTls[Int](uri.authority.host.address())
```

Comment:
weird, but that's how the scheme is stored in akka http... maybe they have a bug...